mitodl/ocw-hugo-themes
# course resources revisions

#### Pre-Flight checklist


#### What are the relevant tickets?
No Ticket :) but this stuff was discussed in this [thread](https://github.com/mitodl/hq/discussions/92#discussioncomment-3028326)

#### What's this PR do?
- Switches taxonomy templates (after renaming `terms.html` to `term.html`). For context: [View this](https://discourse.gohugo.io/t/taxonomy-html-and-terms-html-reversed/39234). All thanks to @gumaerc!
- Adjusts resource title so it looks fine on smaller screens
- Hides/Does not render the learning resource type which doesn't have any resource
- Checks if all learning resource types have no resources then print a single message "No Resources found."
- Some more little tweaks.

#### How should this be manually tested?
- Checkout this branch.
- Verify all the above-mentioned points.
- Verify that all things work as expected. (Some testing instructions are mentioned in the description of this [PR](https://github.com/mitodl/ocw-hugo-themes/pull/753))
- Any use-case that needs to be tested.

#### Screenshots
<img width="385" alt="image" src="https://user-images.githubusercontent.com/93309234/176184311-1e29167a-6c4d-4ffb-9e8a-75c90f3430f2.png">

<img width="678" alt="image" src="https://user-images.githubusercontent.com/93309234/176183943-9db847ca-88a8-4d8a-b673-0f2ecfe7b837.png">
